linode/manager
# M3-1887 release and auto changelog scripts POC

**Purpose**
A python script that is more flexible than git auto changelog, searches for a list of keywords in the commit title rather than strict `add: , fix:` ect., also the script will not consider add tests or other automation commits to changelog. Additionally, a script that can generate a release candidate, bump package.json version, generate changelog, delete previous release branch, and push release candidate. 

**To Test**
To test just the changelog script run `python ~/manager/generate_changelog.py #.##.# yyyy.MM.dd origin` the first parameter is the release version, second date, third name of your manager remote repo. Then check for changelog updates.
